.circleci/config.yml

```diff
@@ -62,8 +62,13 @@ jobs:
           environment:
             JAVA_HOME: /usr/lib/jvm/default-java
           command: |
-            make rsthtml
+            make rsthtml SPHINXOPTS="-b html -W --keep-going -n -T"
+            pydocs=$?
             make javadocs
+            javadocs=$?
+            err=$((pydocs + javadocs))
+            echo "Build test result: $err"
+            test $err = 0
```

Comment:
Why do we need these lines?

Reply:
great point. Setting +e would be required, but messing around with inducing failures in local I don't see any benefit. Reverting these changes and I'll push the modifications to the Makefile :)